s3-sqs-lambda-terraform/example-pattern.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "AWS Amazon S3 Lambda with SQS queue",
+  "description": "Create a Lambda function triggered for every jpg image file stored to S3 via SQS.",
+  "language": "node.js",
+  "level": "200",
+  "framework": "Terraform",
+  "introBox": {
+    "headline": "How to create an EventBridge rule with S3 as the event source and Step Functions as target.",
+    "text": [
+      "The terraform manifest deploys a Lambda function, an SQS queue, one S3 bucket and the IAM resources required to run the application.",
+      "An SQS queue consumes ObjectCreated events from an Amazon S3 bucket if the file has .jpg extension. The SQS triggers a Lambda function.",
+      "The Lambda app acts as a boilerplate code.",
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-sqs-lambda-terraform",
+      "templateURL": "serverless-patterns/s3-sqs-lambda-terraform",
+      "projectFolder": "s3-sqs-lambda-terraform",
+      "templateFile": "main.tf"
+    }
+  },
+  "resources": {
+    "bullets": [
+      {
+        "text": "Building resilient serverless patterns by combining messaging services",
+        "link": "https://github.com/aws-samples/s3-to-lambda-patterns"
```

Comment:
Link is outdated and doesn't really match the text

Reply:
Thanks for the heads up, reference updated